lib/ansible/modules/source_control/subversion.py

```diff
@@ -132,12 +140,19 @@ def _exec(self, args, check_rc=True):
             '--trust-server-cert',
             '--no-auth-cache',
         ]
+        stdin_data = None
         if self.username:
             bits.extend(["--username", self.username])
         if self.password:
-            bits.extend(["--password", self.password])
+            if self.has_option_password_from_stdin():
+                bits.extend(["--password-from-stdin"])
+                stdin_data = self.password
+            else:
+                self.module.warn("The authentication provided will be used on the svn command line and is not secure. "
```

Comment:
```suggestion
                self.module.warn("The password provided will be used on the svn command line and is not secure. "
```

Reply:
We censor anything including the word password, so I had to avoid it.

Reply:
Doh! That's funny.